applications/luci-app-hd-idle/root/usr/share/rpcd/acl.d/luci-app-hd-idle.json

```diff
@@ -3,7 +3,8 @@
 		"description": "Grant UCI access for luci-app-hd-idle",
 		"read": {
 			"file": {
-				"/dev": [ "list" ]
+				"/dev": [ "list" ],
```

Comment:
Since you removed the `fs.list('/dev')` call you can most likely drop the `"/dev": [ "list" ],` permission as well.